### PR TITLE
Fix multi db support

### PR DESCRIPTION
**Pull request by alexgleith**
* This should fix the case where multiple DBs are being used in the one SQS queue

Closes #32

### CODE REVIEW

File: odc/stats/tasks.py [hunk 1]
```diff
@@ -544,14 +543,22 @@ def stream_from_sqs(
...
-            else: # if it is the test_tiles.db, we load it from local
+            else:
+                # If it is the test_tiles.db, we load it from local
                 key = filedb
             local_cache_file = key.split("/")[-1]
-            if not os.path.isfile(local_cache_file):  # use the download filedb from S3 as the init context flag
+            # Use the downloaded filedb from S3 as the init context flag
+            if not os.path.isfile(local_cache_file):
                 self.init_from_sqs(filedb)
+            else:
```
Comment (supermarkion) [inline odc/stats/tasks.py:551]: My original fix plan is: 

No matter we have the db_cache file locally or not, we will always run `self.init_from_sqs(filedb)`. But we do the if-else check on this https://github.com/opendatacube/odc-stats/blob/develop/odc/stats/tasks.py#L383 to not download same db_cache file twice.

Of course, your version also works.

**Review by supermarkion — APPROVED** [review]
Looks all good to me.

File: odc/stats/tasks.py (hunk 1 above)
Comment (alexgleith) [reply odc/stats/tasks.py:551]: I think mine is simpler... but yours updates those other fields, so, I think yours is safer.

I'll change this PR.

**Review by supermarkion — APPROVED** [review]
Looks very good to me!

**Review by supermarkion — APPROVED** [review]
All good. Cheers

**Pull request merged**